api/server/server.go

```diff
@@ -126,7 +126,8 @@ func (s *Server) makeHTTPHandler(handler httputils.APIFunc) http.HandlerFunc {
 		// apply to all requests. Data that is specific to the
 		// immediate function being called should still be passed
 		// as 'args' on the function call.
-		ctx := context.WithValue(context.Background(), dockerversion.UAStringKey, r.Header.Get("User-Agent"))
+		type key string
+		ctx := context.WithValue(context.Background(), key(dockerversion.UAStringKey), r.Header.Get("User-Agent"))
```

Comment:
This breaks https://github.com/moby/moby/blob/9aac187ce1592249ab5f1556d362f2b48ff9c671/dockerversion/useragent.go#L42

Reply:
Ahm, I recall doing it there now 😞